app/Http/Controllers/BeatmapsController.php

```diff
@@ -298,8 +300,9 @@ public function scores($id)
 
             static $scoreIncludes = ['user', 'user.country', 'user.cover'];
 
+            $scoreTransformer = is_api_request() ? new ScoreTransformer() : new SoloScoreTransformer();
             $results = [
-                'scores' => json_collection($query->visibleUsers()->forListing(), 'Score', $scoreIncludes),
+                'scores' => json_collection($query->visibleUsers()->forListing(), $scoreTransformer, $scoreIncludes),
```

Comment:
transformer for `$userScore` also needs to be updated, same for `userScores` method.